contracts/chainlink/ChainLink.sol

```diff
@@ -8,52 +8,59 @@ contract ChainLink is ChainlinkClient, Oracle {
     ERC20 link;
     uint256 maxr; // Max reward
 
+    bytes32 public lastQueryId;
+
+    mapping(bytes32 => bytes32) linkrs;
```

Comment:
Add `linkrs` as a mapping of the LINK `jobId` (`bytes32`) to Bitcoin price `jobId`.